fitdata_test.py

```diff
@@ -38,9 +38,7 @@
 residual,best_shift = hermitian_function_test(s[
     'ph2',coh_sel['ph2']]['ph1',coh_sel['ph1']])
 print("best shift is",best_shift)
-s.ft('t2')
-s *= exp(1j*2*pi*best_shift*s.fromaxis('t2'))
-s.ift('t2')
+s.setaxis('t2', lambda x: x-best_shift).register_axis({'t2':0})
```

Comment:
I made this change in several places.  It should be equivalent.  If it looks like there is a phasing error, then add `best_shift` rather than subtracting (though this matches code I had used elsewhere that works.)

Functionally, this is equivalent to what you had before, but uses some of the pySpecData features for Fourier transforms